docs/server/source/server-reference/bigchaindb-cli.md

```diff
@@ -119,13 +124,14 @@ If the command succeeds, it will create an election and return an `election_id`.
 **NOTE**: The election proposal consists of vote tokens allocated to each current validator as per their voting power. Validators then cast their votes to approve the change to the validator set by spending their vote tokens.
 
 
-#### upsert-validator approve
+#### election approve
+
+Approve an election by voting for it. The propsal generated by executing `bigchaindb election new ...` can approved by the validators using this command. The validator who is approving the proposal will spend all their votes i.e. if the validator has a network power of `10` then they will cast `10` votes for the proposal.
```

Comment:
`can approved by ...`  -> `can be approved by ...`

Reply:
Fixed